### ISSUE TITLE
[Bug]: driver.get_video_events returns an empty array

### ISSUE THREAD

**Issue by CornholioLU** [bropat/eufy-security-ws#119]
### Version

0.9.3

### Node version

16.16.0

### Operating System type

Docker

### Operating system version

Alpine

### Describe the bug

The command driver.get_video_events returns an empty array:

```
{
    "type": "result",
    "success": true,
    "result": {
        "events": []
    }
}
```

Other commands (start_listening for example) work as expected.

### To reproduce

1. Start docker image
2. Connect via Postman to the ws:// endpoint and send:
  - {"command": "set_api_schema", "schemaVersion": 11 }
  - {"command": "start_listening" }
  - {"command": "driver.get_video_events" }

### Screenshots & Logfiles

```
2022-07-18 17:21:39.679  DEBUG [HTTPApi.request] Request: 
{
  method: 'post',
  endpoint: 'v1/house/event/video_list',
  token: '5582f0e988c6107c52d00bd44e468ab6c691790296a05e95',
  data: {
    device_sn: '',
    end_time: 1658164899,
    id: 0,
    id_type: 1,
    is_favorite: false,
    num: 1000,
    pullup: true,
    shared: true,
    start_time: 1185124899,
    station_sn: '',
    storage: 0,
    transaction: '1658164899678'
  }
}
2022-07-18 17:21:39.770  DEBUG [HTTPApi.request] Response: 
{
  response: {
    code: 10000,
    msg: 'Failed to request.'
  }
}
2022-07-18 17:21:39.771  DEBUG [HTTPApi._getEvents] getVideoEvents - Response: 
{
  code: 10000,
  msg: 'Failed to request.'
}
2022-07-18 17:21:39.772  ERROR [HTTPApi._getEvents] getVideoEvents - Response code not ok 
{
  code: 10000,
  msg: 'Failed to request.'
}

```

**Comment by winstona** [comment]
I am seeing this same issue as well. In my case this is something new I'm working on - so I cannot confirm that it previously had been working and now is in a broken state.

**Comment by bropat** [comment]
Checked and confirm the problem.
I have not finished revising this part since the "new" Eufy Api that introduced the houses...
I'm sorry. Will be fixed in the next release.

**Comment by bropat** [comment]
https://github.com/bropat/eufy-security-client/issues/186

**Comment by bropat** [comment]
Fixed in release 0.9.4
